pkg/aws_keyhub/aws.go

```diff
@@ -37,7 +38,12 @@ func StsAssumeRoleWithSAML(context context.Context, principalArn string, roleArn
 }
 
 func VerifyIfLoginWasSuccessful(context context.Context, profile string, roleArn string) {
-	cfg, err := config.LoadDefaultConfig(context, config.WithSharedConfigProfile(profile))
+	// There is not fallback on default profile, this might be a bug in aws go v2 sdk. For now set default region to avoid errors.
```

Comment:
Hier graag een link naar onze ticket:
```suggestion
	// There is no fallback on default profile, this might be a bug in aws go v2 sdk. For now set default region to avoid errors. See CLOUD-3247
```

Reply:
I'm not sure if we should commit internal Jira Issue Ticket ID's in a public project.

Reply:
o yes. This is a public repository.